public/app/features/storage/StoragePage.tsx

```diff
@@ -163,9 +163,9 @@ export default function StoragePage(props: Props) {
 
     return (
       <div className={styles.wrapper}>
-        <HorizontalGroup width="100%" justify="space-between" spacing={'md'} height={25}>
```

Comment:
@grafana/grafana-frontend-platform is there a replacement for the `width` and `height` props?

Reply:
We haven't added those props to the Stack component yet, but you can use Box for this: `<Box display='flex' justifyContent='space-between' width='100%' height={3} />`. Note that height is 8-based, i.e., the value is multiplied by 8 to produce pixel value.

Reply:
@Clarity-89 thanks! updated in [dfb7491](https://github.com/grafana/grafana/pull/86888/commits/dfb7491ebfe028c643acfed5c1f382bf30cfa3ce)